examples/demo/src/types.ts

```diff
@@ -87,6 +94,27 @@ export interface ResourceMatch {
 
 type FilterClassKey = 'button' | 'form';
 
+export interface ToolbarProps<T extends Record = Record> {
+    handleSubmitWithRedirect: (redirect: RedirectionSideEffect) => void;
```

Comment:
all these props are injected by React-admin, so they must be marked as optional.